src/app/find-help/search-result/search-result.component.html

```diff
@@ -1,4 +1,4 @@
-<mat-card class="result">
+<mat-card class="result" (click)="onViewOffer()">
```

Comment:
This should have a proper link in `<a>` and with a `href` attribute, perhaps generated by the router.

Would it be possible to wrap the whole card contents with `<a>` or - if providing a link here is impractical at the moment - a `<button>`?

Reply:
I put all in a but without href